### PR TITLE
Helm katacoda update

### PR DESCRIPTION
**Pull request by craigtyk**

### CODE REVIEW

**Review by marksou — APPROVED** [review]
LGTM

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md [hunk 1]
```diff
@@ -30,30 +31,44 @@ You can also find the *Tyk OSS* Helm chart in [artifacthub](https://artifacthub.
...
 
-### Add Tyk official Helm repo
+### Interactive tutorial
+Get hands-on practice deploying a simple configuration within a live environment!
+<div id="tyk-install-oss-k8s-helm"
+    data-katacoda-id="tyk-education/tyk-install-oss-k8s-helm"
+    data-katacoda-color="00d9ba"
+    style="height: 600px; padding-top: 20px;">
+</div>
+You can find full configuration details in the steps below.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md]: This row is not showing up properly 
image

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md [hunk 2]
```diff
@@ -30,30 +31,44 @@ You can also find the *Tyk OSS* Helm chart in [artifacthub](https://artifacthub.
...
+
+### Step 1 - Add Tyk official Helm repo
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:44]: this title starts straight after the interactive tutorial. Maybe, for clarity, we can add another level `### Step by step installation guild`  and then put the title of each step with heading level 4, `####` so if someone breeze through the page he should understand its structure.

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md [hunk 3]
```diff
@@ -30,30 +31,44 @@ You can also find the *Tyk OSS* Helm chart in [artifacthub](https://artifacthub.
 If you are interested in contributing, suggesting changes or creating PRs, please use our 
 [GitHub repo](https://github.com/TykTechnologies/tyk-helm-chart/tree/master/tyk-headless).
 
-### Add Tyk official Helm repo
+### Interactive tutorial
+Get hands-on practice deploying a simple configuration within a live environment!
+<div id="tyk-install-oss-k8s-helm"
+    data-katacoda-id="tyk-education/tyk-install-oss-k8s-helm"
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:31]: will be good to change to bitnami Redis in our tutorial, whenever it's possible

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md [hunk 4]
```diff
@@ -30,30 +31,44 @@ You can also find the *Tyk OSS* Helm chart in [artifacthub](https://artifacthub.
 If you are interested in contributing, suggesting changes or creating PRs, please use our 
 [GitHub repo](https://github.com/TykTechnologies/tyk-helm-chart/tree/master/tyk-headless).
 
-### Add Tyk official Helm repo
+### Interactive tutorial
+Get hands-on practice deploying a simple configuration within a live environment!
+<div id="tyk-install-oss-k8s-helm"
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:30]: is there a setting for `open in a separate window`?

Comment (craigtyk) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:30]: Not for that specific session, the workaround would be to add a button that opened a new session in another window.

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md (hunk 1 above)
Comment (craigtyk) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md]: Good catch, this wasn't visible on my side. Will update

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md (hunk 2 above)
Comment (craigtyk) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:44]: aim was to have the tutorial under the installation header as a pre-step to follow along. If you think it's clearer for users, we can move this under it's own lv 2 header and keep only the steps under Installation

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md (hunk 1 above)
Comment (craigtyk) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md]: done

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md (hunk 2 above)
Comment (craigtyk) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:44]: done

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md (hunk 3 above)
Comment (craigtyk) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:31]: agreed. noted to backlog for next version

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md (hunk 4 above)
Comment (craigtyk) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:30]: impacts google analytics for the tutorial. will publish without for now but added to backlog for next update

Comment (letzya) [reply tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:30]: thanks

**Review by letzya — APPROVED** [review]

**Pull request merged**